webpack/dev.config.js

```diff
@@ -8,7 +8,7 @@ const customPath = path.join(__dirname, './customPublicPath');
 const hotScript = 'webpack-hot-middleware/client?path=/__webpack_hmr&dynamicPublicPath=true';
 
 const baseDevConfig = () => ({
-  devtool: 'eval-cheap-module-source-map',
+  devtool: 'eval-source-map',
```

Comment:
<!--what rebuild's are you guys planning? devtool: 'eval-source-map' is good for rebuilds right? Why not use devtool: "eval-cheap-module-source-map"? this would provide rebuild much faster speed in my opinion.-->